scripts/release/build_release_and_tar.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e
+
+# Accept ICICLE_VERSION, ICICLE_OS, and ICICLE_CUDA_VERSION as inputs or use defaults
+ICICLE_VERSION=${1:-icicle30}      # Default to "icicle30" if not set
+ICICLE_OS=${2:-unknown_os}              # Default to "unknown_os" if not set
+ICICLE_CUDA_VERSION=${3:-cuda_unknown} # Default to "cuda_unknown" if not set
+
+# List of fields and curves
+fields=("babybear" "stark252")
```

Comment:
```suggestion
fields=("babybear" "stark252", "m31")
```

Reply:
I wonder if there is a way to make this list public, to not update it everywhere across Icicle when a new field is added

Reply:
I can parse 'fields_and_curves.cmake`